umbraco/Umbraco-CMS
# v9: Use localization keys instead of hardcoded string

In the hacktober spirit i have created a helpful PR, that hopefully fixes some of my mistakes 🙈 

# Notes
- System information copy button will now use localization instead of hardcoded string
- System information will now correctly check if it copied the data correctly to the clipboard
- System information will now only copy to clipboard if it copied data correctly

# How to test
- run `npm install` & `npm run build` to build the frontend
- Run the umbraco installation
- Change language to danish
- Open the help pane and press the copy system information button, it should now correctly display the message in the Danish language(you can add your own translations and test other languages)
